src/index.html

```diff
@@ -5,8 +5,291 @@
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smarphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops" class="nav__link"
+            data-qa="nav-hover">Laptops & computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets"  class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo"  class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac 27 Retina"
+          class="card__img"
+          >
+        <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rate-reviews">Reviews: 5</div>
+        </div>
+        <div class="price">
+          <p class="price__label">Price:</p>
+          <p class="price__value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+          >Buy </a>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac 27 Retina"
+          class="card__img"
+          >
+        <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rate-reviews">Reviews: 5</div>
+        </div>
+        <div class="price">
+          <p class="price__label">Price:</p>
+
+          <p class="price__value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
```

Comment:
Same thing here, only first card should have this data-qa

Reply:
Make sure to remove it from the other cards ;)

Reply:
Done.